### PR TITLE
chore(DEV-1870): Align stats button to the left

### PR DESCRIPTION
**Pull request by rousspb**
Align the stats button to the left

## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

**Comment by EspressiveGithub** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  **packages/cascara/src/components/Dashboard/DashboardDX.fixture.js**: This fixture has been changed.
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7e6d6e4ee954ada7c73934280dc2856eb4399ace
</p>

**Review by andresaraujo — DISMISSED** [review]

**Review by andresaraujo — APPROVED** [review]

**Review by ricardogonz — COMMENTED** [review]
LGTM

**Review by mementablog — APPROVED** [review]
LGTM

**Review by ricardogonz — APPROVED** [review]

File: packages/cascara/src/components/Dashboard/DashboardDX.fixture.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ const WIDGETS = [
   },
 ];
```
Comment (ichbinedgar) [inline packages/cascara/src/components/Dashboard/DashboardDX.fixture.js:82]: LGTM

**Review by ichbinedgar — APPROVED** [review]
LGTM

**Pull request merged**